### ISSUE TITLE
Premature sagging cheeks

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7480]
**Preferred term label:**
Premature sagging cheeks

**Synonyms**
Premature facial sagging

**Definition (free text, please give PubMed ID)**
Drooping or sinking of tissues of the cheeks more than would be expected at a given age. Sagging can occur due to a relative excess of skin and/or lack of elastic recoil as well as fat accumulation.PMID:19338976

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 619451 ;  PMID: 33991472  7/8

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
